pkgs/by-name/zo/zoom-us/package.nix

```diff
@@ -34,13 +64,22 @@ let
     # Then build `zoom-us.unpacked`:
     # `autoPatchelfHook` will report missing library files.
 
-    installPhase = ''
-      runHook preInstall
-      mkdir $out
-      tar -C $out -xf $src
-      mv $out/usr/* $out/
-      runHook postInstall
-    '';
+    installPhase =
+      if stdenv.hostPlatform.isLinux then
+        ''
+          runHook preInstall
+          mkdir $out
+          tar -C $out -xf $src
+          mv $out/usr/* $out/
+          runHook postInstall
+        ''
+      else
+        ''
+          runHook preInstall
+          mkdir -p $out/Applications
+          cp -R *.app $out/Applications/
+          runHook postInstall
+        '';
 
```

Comment:
Previous darwin version had a program `$out/bin/zoom`, created by `makeWrapper`.  Suggestion: Should we reinstate it?

Reply:
I am not sure. On Mac, one doesn't, in general, carry `/bin/*` files in UI bundle packages.